README.md

```
@@ -83,12 +83,36 @@ Online demo: https://tooltip-react-component.vercel.app/
 | mouseLeaveDelay     | number                                                             | 0.1                 | delay time (in second) before tooltip hides when mouse leave                                                                                                     |
 | getTooltipContainer | (triggerNode: HTMLElement) => HTMLElement                          | () => document.body | get container of tooltip, default to body                                                                                                                        |
 | destroyOnHidden     | boolean                                                            | false               | destroy tooltip when it is hidden                                                                                                                                |
-| align               | object                                                             |                     | align config of tooltip. Please ref demo for usage example                                                                                                       |
+| align               | object                                                             |                     | align config of tooltip, ref [dom-align](https://github.com/yiminghe/dom-align). See [align overflow](#align-overflow) below                                     |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)